grunt-configs/shell.js

```diff
@@ -24,13 +24,31 @@ module.exports = function(grunt, options) {
                      'static/abtests.json'
         },
 
-        jspmFaciaTool: {
+        jspmInstallFaciaTool: {
             command: 'node ../../node_modules/jspm/jspm.js install',
             options: {
                 execOptions: {
                     cwd: 'facia-tool/public'
                 }
             }
+        },
+
+        jspmInstallStatic: {
+            command: './node_modules/.bin/jspm install',
```

Comment:
I actually received a rate limit error on the first build of this branch. :unamused: 

https://teamcity.gutools.co.uk/viewType.html?buildTypeId=bt1324&branch_Frontend=8688&tab=buildTypeStatusDiv

I think this is worth fixing now by adding the GH credentials to TC. Any advisable steps, @sndrs @theefer @phamann? (AKA How?)